examples/quickstart/jupyter-notebooks/sql-tutorial.ipynb

```diff
@@ -0,0 +1,765 @@
+    "Check the objects' `DATA_TYPE` property for SQL data types. You should see TIMESTAMP, BIGINT, and VARCHAR SQL data types. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9227d6c-1d8c-4169-b13b-a08625c4011f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "  \"query\":\"SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE \\\"TABLE_SCHEMA\\\" = 'druid' AND \\\"TABLE_NAME\\\" = 'wikipedia-sql-tutorial' LIMIT 7\"\n",
+    "})\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "\n",
+    "print(\"\\033[1mQuery\\033[0m:\\n\" + payload)\n",
+    "print(\"\\n\\033[1mResponse\\033[0m: \\n\" + json.dumps(response.json(), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c59ca797-dd91-442b-8d02-67b711b3fcc6",
+   "metadata": {},
+   "source": [
+    "Druid natively interprets VARCHAR as STRING and BIGINT and TIMESTAMP SQL data types as LONG. For reference on how SQL data types map onto Druid native types, see [Standard types](https://druid.apache.org/docs/latest/querying/sql-data-types.html#standard-types).\n",
+    "\n",
+    "### Timestamp values\n",
+    "\n",
+    "Druid stores timestamp values as the number of milliseconds since the Unix epoch.\n",
+    "Primary timestamps are stored in a column named `__time`.\n",
+    "If a dataset doesn't have a timestamp, Druid uses the default value of `1970-01-01 00:00:00`.\n",
+    "\n",
+    "Druid time functions perform best when used with the `__time` column.\n",
+    "By default, time functions use the UTC time zone.\n",
+    "For more information about timestamp handling, see [Date and time functions](https://druid.apache.org/docs/latest/querying/sql-scalar.html#date-and-time-functions).\n",
+    "\n",
+    "Run the following cell to see Druid SQL `TIME_IN_INTERVAL` scalar function at work. This query checks whether a timestamp is contained within a specified interval. The results are grouped by date."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f7e3d62a-1325-4992-8bcd-c0f1925704bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "endpoint = \"/druid/v2/sql\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "  \"query\":\"SELECT TIME_IN_INTERVAL(__time, '2016-06-27T12:00:00/2016-06-27T13:00:00'), COUNT(*) FROM \\\"wikipedia-sql-tutorial\\\" \\\n",
```

Comment:
as a general practice, let's alias our columns so that the output is more informative. The current output is this:
```
Response: 
[
    {
        "EXPR$0": false,
        "EXPR$1": 23214
    },
    {
        "EXPR$0": true,
        "EXPR$1": 1219
    }
]
```
You can change the SELEXT to alias the columns:
```
SELECT TIME_IN_INTERVAL(__time, '2016-06-27T12:00:00/2016-06-27T13:00:00') AS existing, COUNT(*) AS total 
```
Then the output has column aliases like this:
```
Response: 
[
    {
        "existing": false,
        "total": 23214
    },
    {
        "existing": true,
        "total": 1219
    }
]
```

Reply:
I've changed the code example to a slightly different query.